bin/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'item'
+class MusicAlbum < Item
+  attr_accessor :on_spotify, :name
+  attr_reader :publish_date
+
+  def initialize(name, publish_date, on_spotify)
+    super(publish_date)
+    @name = name
+    @publised_date = publised_date
```

Comment:
the super() constructor is already setting this property, you don't need to set it again. Also, please, notice the typo in this line and in the parameters, it should be publish_date according to the diagram.